### DISCUSSION TITLE
Copilot is wrong when i work with Elixir

### DISCUSSION THREAD

**Discussion by DougNeo** [community/community#11023] [Copilot Conversations]
Good Afternoon!

I has use GitHub Copilot for coding in Elixir and Ruby in my machine, but the copilot does't undestandig when i coding in Elixir or Ruby beacause the copilot is suggest code ruby for elixir, often is wrong. Is this a bug because language and synthax is a similar? Does anyone else have the problem I have?

PS: Sorry for my english.

**Comment by jbosse** [comment]
Yes, I think because Elixir is syntactically similar to Ruby CoPilot often suggest Ruby when writing Elixir.

**Comment by derekpappas** [comment]
Does copliot read the language type in a comment as a hint to the AI?

/* copilot_language = elixir */

Does copilot know the language types in the output?

Is there more than one suggestion with an associated language type in the AI output and if so can the AI output  be post processed to select the correct language based on the selected language in the associated comment example?
